src/components/Form/CalendarInput/CalendarInput.js

```diff
@@ -197,23 +198,45 @@ CalendarInput.defaultProps = {
 };
 
 CalendarInput.propTypes = {
+    /** If true ,there is a "x" mark inside the component,
+     *  if "x" mark is clicked the element inside the component will be cleared,
+     *  if false there is no "x" mark in the component.*/
     clearable: PropTypes.bool,
-    dateFormat: PropTypes.oneOfType([
-        PropTypes.string,
-        PropTypes.bool
-    ]),
+
+    /** If True, users won't be able to interact with the component*/
     disabled: PropTypes.bool,
+
     errors: PropTypes.objectOf(
         PropTypes.arrayOf(PropTypes.string)
     ),
+
+    /** Layout that arranges views of the component*/
     gridClass: PropTypes.string,
+
+    /** Specifies the icon class to be displayed within the component*/
     iconClass: PropTypes.string,
+
+    /** It calls a functions that manipulate the selectable date
```

Comment:
R: "Specify a callback to determine the dates that can be selected. The function receives (currentDate, selectedDate) and shall return a true or false whether the date is valid or not."